meeting_task/recursion.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+
+</body>
+
+</html>
+<script>
+    var list = [];
+    let n = window.prompt("Enter Value");
+
+    function prime(n) {
+        if (n <= 2) {
```

Comment:
I think 2 is prime
https://en.wikipedia.org/wiki/Prime_number

Reply:
ok sir i will  correct it